db/models/__init__.py

```diff
@@ -126,3 +126,5 @@ class InterfaceDetailDoc(DocumentBaseModel):
 
     available_tags: Dict[str, List[List[int]]]
     tag_ranges: Dict[str, List[List[int]]]
+    special_available_tags: Dict[str, List[str]]
+    special_tag_range: Dict[str, List[str]]
```

Comment:
Looks good, but now thinking again about the name `special_tag_range`, I believe I can lead to confusion, in our code base a tag range represents really an inclusive-inclusive list of two integers, where here with `special_tag_range` the underlying values are just a list of special values and not a range. How should we name it to avoid confusion? just `special_tags`? Let me know if you also have a better suggestion that reflects its purpose and use case.

Reply:
Name changed to `special_tags`